### PR TITLE
docs(stitcher): Add Video Stitcher samples and tests

### PR DESCRIPTION
**Pull request by irataxy**
b:202972981

Part 1 of 4: Slates (see https://cloud.google.com/video-stitcher/docs/how-to/managing-slates) 

**Please enable the API in the test project**

### CODE REVIEW

File: google-cloud-video-stitcher/samples/Gemfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+source "https://rubygems.org"
+
+master = ENV["GOOGLE_CLOUD_SAMPLES_TEST"] == "master"
+
+gem "google-cloud-video-stitcher", path: master ? "../../google-cloud-video-stitcher" : nil
+
+group :test do
+  gem "google-style", "~> 1.25.1"
+  gem "minitest", "~> 5.14"
```
Comment (dazuma) [inline google-cloud-video-stitcher/samples/Gemfile:23]: Would you mind using `"google-style", "~> 1.26.1"` and `"minitest", "~> 5.16"`? We're trying to get everything updated to those versions.

File: google-cloud-video-stitcher/samples/Gemfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "https://rubygems.org"
+
+master = ENV["GOOGLE_CLOUD_SAMPLES_TEST"] == "master"
+
+gem "google-cloud-video-stitcher", path: master ? "../../google-cloud-video-stitcher" : nil
```
Comment (dazuma) [inline google-cloud-video-stitcher/samples/Gemfile:19]: Could you also insert:

```ruby
gem "google-cloud-video-stitcher-v1", path: master ? "../../google-cloud-video-stitcher-v1" : nil
```

File: google-cloud-video-stitcher/samples/Rakefile [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Google LLC
```
Comment (dazuma) [inline google-cloud-video-stitcher/samples/Rakefile:1]: 2022

File: google-cloud-video-stitcher/samples/Gemfile [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Google, Inc
```
Comment (dazuma) [inline google-cloud-video-stitcher/samples/Gemfile:1]: Use `Google LLC` rather than `Google, Inc`. Likewise for several of the other files.

**Review by dazuma — CHANGES_REQUESTED** [review]

File: google-cloud-video-stitcher/samples/Gemfile (hunk 4 above)
Comment (irataxy) [reply google-cloud-video-stitcher/samples/Gemfile]: Done

File: google-cloud-video-stitcher/samples/Rakefile (hunk 3 above)
Comment (irataxy) [reply google-cloud-video-stitcher/samples/Rakefile]: Done

File: google-cloud-video-stitcher/samples/Gemfile (hunk 2 above)
Comment (irataxy) [reply google-cloud-video-stitcher/samples/Gemfile:19]: Done

File: google-cloud-video-stitcher/samples/Gemfile (hunk 1 above)
Comment (irataxy) [reply google-cloud-video-stitcher/samples/Gemfile]: Done

**Review by dazuma — APPROVED** [review]

**Pull request merged**